frontend/tsconfig.json

```diff
@@ -15,7 +15,7 @@
 
     /* Linting */
     "strict": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
this was needed as per this awesome tutorial: https://johnnyreilly.com/typescript-eslint-no-unused-vars